content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+    {{< /text >}}
+
+1.  Create 2 GKE clusters for use with the multicluster feature.  _Note:_ `--enable-ip-alias` is required to
+    allow inter-cluster direct pod-to-pod communication.  The `zone` value must be one of the
+    [GCP zones](https://cloud.google.com/compute/docs/regions-zones/).
+
+    {{< text bash >}}
+    $ zone="us-east1-b"
+    $ cluster="cluster-1"
+    $ gcloud container clusters create $cluster --zone $zone --username "admin" \
+      --cluster-version "1.9.6-gke.1" --machine-type "n1-standard-2" --image-type "COS" --disk-size "100" \
+      --scopes "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.read_only",\
+      "https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring",\
+      "https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly",\
+      "https://www.googleapis.com/auth/trace.append" \
+      --num-nodes "4" --network "default" --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --async
+    $ cluster="cluster-2"
+    $ gcloud container clusters create $cluster --zone $zone --username "admin" \
+      --cluster-version "1.9.6-gke.1" --machine-type "n1-standard-2" --image-type "COS" --disk-size "100" \
+      --scopes "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.read_only",\
+      "https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring",\
+      "https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly",\
+      "https://www.googleapis.com/auth/trace.append" \
+      --num-nodes "4" --network "default" --enable-cloud-logging --enable-cloud-monitoring --enable-ip-alias --async
+    {{< /text >}}
+
+1.  Wait for clusters to transition to `RUNNING` state:
+
+    {{< text bash >}}
+    $ gcloud container clusters list
+    {{< /text >}}
+
+1.  Get the clusters' credentials (by default stored in `~/.kube/config` or in the file the KUBECONFIG environment
+    variable is set to).
+
+    {{< text bash >}}
+    $ gcloud container clusters get-credentials cluster-1 --zone $zone
+    $ gcloud container clusters get-credentials cluster-2 --zone $zone
+    {{< /text >}}
+
+1.  Validate `kubectl` access to each cluster
+
+    1.  Check cluster-1
+
+        {{< text bash >}}
+        $ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Check cluster-2
+
+        {{< text bash >}}
+        $ kubectl config use-context "gke_${proj}_${zone}_cluster-2"
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+1.  Create a `cluster-admin` cluster role binding tied to the kubernetes credentials associated with your GCP user.
+    _NOTE:_ replace `mygcp@gmail.com` with the email tied to your Google Cloud account.
```

Comment:
Elsewhere, you used _Note_, the doc should be consistent.

Reply:
ack